libopenage/audio/dynamic_resource.h

```diff
@@ -39,7 +39,7 @@ struct chunk_info_t {
 	/** The chunk's current state. */
 	std::atomic<state_t> state;
 
-	/** The chunk's actual size. */
+	/** The chunk's actual amount of values in buffer. */
```

Comment:
is that clear? what would you understand from that?

Reply:
It is the number of bytes that are stored in this chunk of audio. Namely the size of `buffer`.

Reply:
What I wanted to say is "it's the number of valid entries the array". An entry is a int16_t. 
in dynamic_resource.cpp you see this in the constructor:
```
size{0},
buffer{std::make_unique<int16_t[]>(buffer_size)} {}
```
size is initialized to zero, because there are no values in the beginning.
the buffer_size argument determines the maximum number of values (not bytes) the buffer can contain (if i understood make_unique correctly).

Reply:
yea that's true, and your (updated) code comment says the right thing :)